### ISSUE TITLE
Search function skipped the first page

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fleszar1/flexpaper#235]
```
What steps will reproduce the problem?
1. load a doc, which contains a keyword appear on every page
2. search that keyword, viewer will go to second page
3. search until the end of doc, after the message box, click search button 
again, it will find and highlight the keyword in the first page

What is the expected output? What do you see instead?
should find keyword in first page at the first search

What version of the product are you using? On what operating system?
1.4.7

Please provide any additional information below.

```

Original issue reported on code.google.com by `barrykwo...@gmail.com` on 24 Nov 2011 at 4:33

**Comment by GoogleCodeExporter** [comment]
```
which page were you looking at when you started your search? the first?
```

Original comment by `erik.eng...@devaldi.com` on 28 Nov 2011 at 1:51

**Comment by GoogleCodeExporter** [comment]
```
yes, i am viewing the first page
search immediately after the document is loaded completely into viewer
```

Original comment by `barrykwo...@gmail.com` on 28 Nov 2011 at 3:19

**Comment by GoogleCodeExporter** [comment]
```
if I go to second page first, and then back to first page, the search still 
cannot find the keyword in first page
it must search until end of document, after displaying the message, and click 
the search button again, it can find the keyword in first page
```

Original comment by `barrykwo...@gmail.com` on 28 Nov 2011 at 3:23

**Comment by GoogleCodeExporter** [comment]
Original comment by `erik.eng...@devaldi.com` on 29 Dec 2011 at 5:19
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 1.5.0
```

Original comment by `erik.eng...@devaldi.com` on 2 Feb 2012 at 4:02
- Changed state: **Fixed**
